src/elm/Session/LoggedIn.elm

```diff
@@ -784,6 +877,11 @@ type Msg
     | CloseCommunitySelector
     | SelectCommunity Symbol (Cmd Msg)
     | HideFeedbackLocal
+    | CloseAddContactModal
+    | EnteredContactOption String
+    | GotContactMsg Contact.Msg
+    | SubmittedContactModalForm
+    | CompletedUpdateContact (Result (Graphql.Http.Error (Maybe Profile.Model)) (Maybe Profile.Model))
```

Comment:
@amiskov can you give us some insight on how to move some of those msgs to `Contact.elm` like you did on #476?

Reply:
I was actually thinking these are more specific to the modal itself, so didn't really try to move them to `Contact.elm`, but on second thought, moving them to `Contact.elm` seems like a good idea.

For `EnteredContactOption`, we should probably provide a way to optionally display a type selector. On the profile view for editing contacts, instead of having a type selector, all of the types are already listed as separate inputs, so I guess depending on whether or not there should be a type selector, we can decide on rendering a type selector or just list all of the types as separate inputs. 

For `CompletedUpdateContact` I guess we could have a `onSubmit` passed to `Contact.init`.

Reply:
Yeah, probably, we can store things related to Contact into its own module, export `Msg` from it and use only `GotContactMsg Contact.Msg` here, in `LoggedIn`.

Also, we can adopt a [remote data](https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData) pattern for the `Result` that `CompletedUpdateContact` accepts. The package for `RemoteData` is already added to the project.

@NeoVier I did something similar in `src/elm/Search.elm`, maybe you can find some insights here. Also, please, make sure you have the latest changes from the `master` branch of the main repo.